Cargo.lock

```diff
@@ -1284,9 +1284,9 @@ checksum = "49874b5167b65d7193b8aba1567f5c7d93d001cafc34600cee003eda787e483f"
 
 [[package]]
 name = "versionize"
-version = "0.1.9"
+version = "0.1.10"
```

Comment:
Do we need to update versionize_derive as well?

Reply:
Nope, only the main versionize crate was updated